JobScheduler/appsettingsjobscheduler.Development.json

```diff
@@ -17,12 +17,12 @@
         "IdentityServerConfig": {
             "IdentityUrl": "https://localhost:5005/",
             "ClientId": "dex-jobscheduler",
-            "ClientSecret": "`HP(@`aQ]>wO4XJa4hU>,-HLHpo4h~"
+            "ClientSecret": "fGe34rE6F53dsT"
         },
         "RabbitMQ": {
             "Hostname": "localhost",
-            "Username": "test",
-            "Password": "test"
+            "Username": "guest",
```

Comment:
Moeten hier geen betere credentials komen m.b.t security of is dat nog een ToDo?

Reply:
This is purely for local development. I don't think it's needed.

Reply:
Good observation. But I think it's correct the way it's right now. This is just some dummy data so that not everyone can see our username and password that are related to a certain hostname.

The actual information will get inserted on the staging and production server.

The guest, guest credentials are specials credentials for RabbitMQ for localhost (for more information check this link out https://www.rabbitmq.com/access-control.html).

@niraymak is this correct / did I forget something?

Reply:
Correct. The developer might has to switch it to "test" / "test" if you are using the RabbitMQ of our docker-compose.